spec/i18n_spec.rb

```diff
@@ -74,6 +74,7 @@ class BaseTask
         { key: 'time.formats.event_timestamp', locales: %i[zh] },
         { key: 'time.formats.full_date', locales: %i[es] }, # format is the same in Spanish and English
         { key: 'time.formats.sms_date' }, # for us date format
+        { key: 'user_mailer.reset_password_instructions.in_person_warning_description_html', locales: %i[es fr zh] },
```

Comment:
Nit: Only if you are updating the code again- a note that this will temporary until translations come in would be nice